.github/workflows/docker-build-and-push-arm64.yaml

```diff
@@ -80,8 +80,7 @@ jobs:
             *.cache-from=type=registry,ref=ghcr.io/${{ github.repository }}-buildcache:${{ matrix.name }}-${{ matrix.platform }}-${{ github.ref_name }}
             *.cache-from=type=registry,ref=ghcr.io/${{ github.repository }}-buildcache:${{ matrix.name }}-${{ matrix.platform }}-main
             *.cache-to=type=registry,ref=ghcr.io/${{ github.repository }}-buildcache:${{ matrix.name }}-${{ matrix.platform }}-${{ github.ref_name }},mode=max
-          tag-suffix: ${{ matrix.additional-tag-suffix }}-${{ matrix.platform }}
-          tag-prefix: ${{ needs.load-env.outputs.rosdistro }}
+          tag-suffix: ${{ matrix.additional-tag-suffix }}
```

Comment:
`matrix.additional-tag-suffix` will be no longer "additional" `tag-suffix`.
Please rename to `matrix.tag-suffix`.